app/views/catalog/_show_sections/_default_contents.html.erb

```diff
@@ -43,7 +35,7 @@
                         <% child.xpath('file').each do |file| %>
                           <li class="file">
                             <span class="label">File</span>
-                            <% if can_manage %>
+                            <% if can? :manage_content, object %>
```

Comment:
If my previous solution were adopted, this might have become `manage_item` instead of `manage_content`.  So, in general, the default should be `manage_item` unless it's clear that `manage_content` is required.  In general, this distinction between the two is important and needs to be resolved.